Version/Version.swift

```diff
@@ -8,10 +8,10 @@
 
 import Foundation
 
-struct Version: Equatable {
+class Version: NSObject, Equatable, NSCoding, Printable {
```

Comment:
It seems currently Swift won't allow a struct to conform to `NSCoding` :disappointed: